cluster/olm/ceph/generate-rook-csv.sh

```diff
@@ -35,6 +35,11 @@ if ! command -v "$yq" >/dev/null && [ ! -f "$yq" ]; then
     exit 1
 fi
 
+if [[ "$("$yq" --version)" != "yq version 3."* ]]; then
+    echo "yq must be version 3" >/dev/stderr
+    exit 1
+fi
+
```

Comment:
It's really challenging to make sure all the various yq commands get modified the right way for yq v4, so I add this check to be double sure we are calling the script with yq v3.

Reply:
Long term, transitioning to v4 for the CSV gen would help but that a different PR for sure.